src/Icon/_variables.scss

```
@@ -0,0 +1,7 @@
+
+// Icons sizes
+
+$icon-inline: .9rem !default;
```

```suggestion
$icon-inline: .8em !default;
```
This way inline icons will size in relation to font size.

---

@abutterworth fixed!